frontend/packages/shared/src/components/GiteaHeader/ThreeDotsMenu/LocalChangesModal/LocalChangesModal.module.css

```
@@ -0,0 +1,13 @@
+.headingWrapper {
+  display: flex;
+  align-items: center;
+  margin-left: var(--fds-spacing-7);
+}
+
+.headingWrapper > :first-child {
+  margin-right: var(--fds-spacing-2);
+}
+
+.icon {
+  font-size: 2rem;
+}
```

Referer ikke disse til samme element?
```suggestion
.icon {
  font-size: 2rem;
  margin-right: var(--fds-spacing-2);
}
```